### ISSUE TITLE
Dataset problem: NorCPM1 historical run goes to 2030.

### ISSUE THREAD

**Issue by ledm** [ESMValGroup/ESMValCore#504]
Someone in Norway is living in the future: the historical run goes up to 2030! On jasmin, the file is
```
/badc/cmip6/data/CMIP6/CMIP/NCC/NorCPM1/historical/r1i1p1f1/Omon/thetao/gn/latest/thetao_Omon_NorCPM1_historical_r1i1p1f1_gn_201901-202912.nc
```
The file itself seems okay, just it shouldn't be in the historical experiment.

**Comment by zklaus** [comment]
We have had feedback from the modelers and they prepared an erratum for this [here](https://errata.es-doc.org/static/view.html?uid=ad9e4213-f254-e930-a143-030a4b0f8a32).

Bottom line: They needed a control experiment for their decadal forecasts and chose to publish this as an extension of the historical experiment. @veyring, is this a good idea? Should we raise it with CMIP and/or ESGF?

**Comment by valeriupredoi** [comment]
Cheers for taking this upstream @zklaus :beer: Good to see our data alerts scheme working and good to see they've responded so fast. Imho this a problem with ESGF accepting something that should defo be renamed, I'll also have a chat with @agstephens and David Hassell see what they reckon

**Comment by valeriupredoi** [comment]
man this is like publishing a book called "Erwin Rommel's Campaign in the Western Desert, 1941-1991" :rofl: 

@agstephens @durack1 what do you guys think? :beer:

**Comment by durack1** [comment]
@taylor13 @sashakames this seems to be something that we will encounter too

**Comment by sashakames** [comment]
QC is indeed hard.  We need very imaginative folks who can suggest all sorts of scenarios... I presume an errata was opened on this?

**Comment by taylor13** [comment]
@zklaus @valeriupredoi @durack1 The post-historical parts of the simulation should have been identified as the "historical-ext" experiment, not the "historical" experiment.  In designing CMIP we knew that for various reasons folks would want to extend the historical runs at least to the present (~2020), but they wouldn't all use the same forcing, so we wanted to distinguish this portion of the historical run from the standard forcing part of the run.  Here's what Eryring et al. (GMD, 2016) say:
___
"For  model  evaluation  and  for  detection  and  attribution studies (the focus of DAMIP) there would be considerable value in extending the CMIP6 historical simulations beyond the nominal 2014 ending date. To include the more recent observations in model evaluation, modelling groups are encouraged to document and apply forcing data sets representing the post-2014 period. For short extensions (up to a few years) it may be acceptable to simply apply forcing from one of  the  future  scenarios  defined  by  ScenarioMIP.  To  distinguish between the portion of the historical period when all models will use the same forcing data sets (i.e. 1850–2014) from the extended period where different data sets might be used, the experiment for 1850–2014 will be labelled historical (esm-hist in the case of the emission-driven run) and the period from 2015 through near-present will likely be labelled historical-ex (esm-hist-ext)."
___
All the CMIP6 experiments are listed in https://github.com/WCRP-CMIP/CMIP6_CVs/blob/master/CMIP6_experiment_id.json (and viewable at: https://wcrp-cmip.github.io/CMIP6_CVs/docs/CMIP6_experiment_id.html ).

So, we know that such simulations were anticipated, but in this case they were misidentified.  This could be corrected, of course, by rewriting the file metadata (and file names) and republishing, but I'm not sure how important this is.

**Comment by IngoBethke** [comment]
Thanks for clarification and suggestions. 

The use of historical-ext seems indeed more appropriate and I must have overlooked this option. I was familiar with CMIP5 historicalExt but did not notice the name change to historical-ext. In hindsight, I should have contacted you at that time instead of resorting to using the historical experiment. 

I will try to correct this, i.e. first publish "historical-ext" and following publish an updated version of "historical" that excludes the period post-2014. It shouldn't be too much work.

**Comment by zklaus** [comment]
Thanks to all of you, @durack1, @sashakames, @taylor13, @IngoBethke!

Regarding @sashakames comment, we are reporting more and more dataset problems upstream, that is to the modeling groups. Some of them are more responsive, others less.

Would it be useful to CC these reports to a more central body?

**Comment by taylor13** [comment]
@zklaus There is work being done (or perhaps it hasn't yet started?  @davidhassell) to simplify and centralize the reporting of apparent errors of this kind through ES-DOCs.  I'll try to find out the prospects are for that.  If it ican be put in place, I think that would serve the purpose.

**Comment by zklaus** [comment]
@taylor13, that sounds promising! Thanks for keeping us posted!

**Comment by valeriupredoi** [comment]
great communication, guys, much appreciated! :beer:

**Comment by valeriupredoi** [comment]
@mattiarighi can we add @davidhassel back to the ESMValTool gitHub project pls? He used to be in but I can't see him part of it anymore (he's cool, he's just three offices down the hallway from me :grin: ) :beer:

**Comment by mattiarighi** [comment]
@axel-lauer is taking care of this, just send him an e-mail with David's contact details.

**Comment by valeriupredoi** [comment]
> @axel-lauer is taking care of this, just send him an e-mail with David's contact details.

Jawohl, danke sehr :beer:

**Comment by IngoBethke** [comment]
The issue has been fixed. See https://errata.es-doc.org/static/view.html?uid=ad9e4213-f254-e930-a143-030a4b0f8a32

**Comment by davidhassell** [comment]
Hi - the work to  simplify and centralize the reporting of apparent errors in CMIP6 datasets has started, and is due, as far as I'm aware in, ~October. It's not a trivial change, as the errata library needs refactoring to deliver the new functionality.

**Comment by taylor13** [comment]
I agree that the change will not be trivial, and I was told by the development team that it might not be available until the end of the year, so October might be too optimistic.  The important thing is that the WGCM and modeling groups all seem supportive of making the change, so we can proceed.
